features/steps/helper.go

```diff
@@ -0,0 +1,15 @@
+package steps
+
+import (
+	"net/http"
+
+	"github.com/maxcnunes/httpfake"
+)
+
+func healthCheckStatusHandle(status int) httpfake.Responder {
+	return func(w http.ResponseWriter, r *http.Request, rh *httpfake.Request) {
+		rh.Lock()
```

Comment:
(question): why do you need this lock and unlock here ? I don't think this is needed but may be i am missing something ?

Reply:
Done